templates/zerver/integrations/redmine.md

```diff
@@ -22,28 +22,31 @@ The Zulip plugin is now registered with Redmine!
 
 ### Global settings
 
-On your {{ settings_html|safe }}, create a new Redmine bot.
+Log into your Redmine instance, click on **Administration** in the top-left
```

Comment:
We generally use Markdown numbered lists for series of steps (the docs have a nice rendering for them); can you add those?